tests/util/kube_utils.go

```diff
@@ -420,3 +513,74 @@ func FetchAndSaveClusterLogs(namespace string, tempDir string) error {
 	}
 	return multiErr
 }
+
+// ForwardedPort contains the information for port-forwarding a single Kubernetes application
+type ForwardedPort struct {
```

Comment:
Let's not use port-forwarding if possible. It's very brittle (`socat` based). Can you create an issue and add a link here.

Reply:
That's fine, I'm actually not using it anyway.  I originally copied `mixer_test.go` and thought I might need it.